src/ft2font_wrapper.cpp

```diff
@@ -664,7 +668,9 @@ static PyObject *PyFT2Font_set_text(PyFT2Font *self, PyObject *args, PyObject *k
         return NULL;
     }
 
-    CALL_CPP("set_text", self->x->set_text(size, &codepoints[0], angle, flags, xys));
+    if (size > 0) {
+        CALL_CPP("set_text", self->x->set_text(size, &codepoints[0], angle, flags, xys));
```

Comment:
So the problem is that `codepoints[0]` is illegal for an empty vector? If we can guarantee a C++11 compiler (probably not?) you could use `codepoints.data()`. Otherwise, I guess something like this would work:

    uint32_t* codepoints_array = NULL;
    if (size > 0) {
        codepoints_array = &codepoints[0];
    }
    CALL_CPP(... codepoints_array, ...));